packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -275,6 +276,18 @@ class MenuAnchor extends StatefulWidget {
   /// {@macro flutter.material.MenuBar.shortcuts_note}
   final List<Widget> menuChildren;
 
+  /// The optional alignment for the widget that this [MenuAnchor] surrounds.
+  ///
+  /// If provided, [builder] is placed inside an [Align] using this alignment.
```

Comment:
Turbo nit: I think "If provided" applies to the builder in this case, but I think your intent is to say that "If [builderAlignment] is provided...."